src/ext.js

```diff
@@ -132,7 +149,89 @@ export default function ext(/* env */) {
                     color: null,
                   },
                 },
-              ],
+                useBackgroundImage: {
+                  ref: 'style.background.isUsed',
+                  type: 'boolean',
+                  translation: 'properties.backgroundImage.use',
+                  component: 'switch',
+                  defaultValue: false,
```

Comment:
@Caele Do you think we will ever have the option to set a background image from the Theme? In that case we cannot set a defaultValue here, but it feels like quite a stretch

Reply:
no that would be strange, the images right now has to come from within sense, you cannot link to an external one (which a theme one would almost be)